examples/express/index.js

```diff
@@ -0,0 +1,159 @@
+'use strict'
+
+// FOR DEVELOPMENT ONLY: allow for self-signed/invalid certificates universally
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+
+// Use Express.js for routing.
+const express = require('express')
+var session = require('express-session')
+
+const app = express()
+
+app.use(session({
+  secret: 'CHANGE-IT',
+  resave: false,
+  saveUninitialized: true,
+  cookie: {}
+}))
+
+// Import
+const helper = require('openid-client-helper')
+
+// // A ForgeRock configuration example.
+// const issuerMetadata = {
+//   issuer: 'https://default.iam.example.com/am/oauth2',
+//   revocation_endpoint: 'https://default.iam.example.com/am/oauth2/token/revoke' // A property not exposed in the OpenID configuration document.
+// }
+// const clientMetadata = {
+//   client_id: 'node-openid-client',
+//   client_secret: 'password',
+//   // token_endpoint_auth_method: 'client_secret_post', // The default is 'client_secret_basic'.
+//   redirect_uri: 'http://localhost:3333/redirect',
+//   post_logout_redirect_uri: 'http://localhost:3333'
+// }
+// const resources = {
+//   'https://default.iam.example.com/openidm/info/login': {
+//     scope: 'fr:idm:profile'
+//   }
+// }
+
+// // A Google configuration example.
+// const issuerMetadata = {
+//   issuer: 'https://accounts.google.com'
+// }
+// const clientMetadata = {
+//   client_id: 'google-client.apps.googleusercontent.com',
+//   client_secret: 'google-client-secret',
+//   // token_endpoint_auth_method: 'client_secret_post', // The default is 'client_secret_basic'.
+//   redirect_uri: 'http://localhost:3333/redirect'
+//   // `Google` does not support RP-initiated logout.
+// }
+// const resources = {
+//   'https://openidconnect.googleapis.com/v1/userinfo': {
+//     scope: 'https://www.googleapis.com/auth/userinfo.profile'
+//   }
+// }
+
+// Abstract configuration example.
+const issuerMetadata = {
+  issuer: 'openid-provider.well-known.openid-configuration-uri'
```

Comment:
I feel like this would be a bit more obvious if it were an actual URL

Reply:
Changed; I replaced it with the abstract URI they used in OIDC docs.